deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -121,6 +123,9 @@ def make_vuln_callback(pattern):
 
 
 def load_db(db_path):
+    """
+    Load a database in db_path
+    """
```

Comment:
For very simple cases, the docstrings should be a one-liner:
```suggestion
    """Load a database in db_path"""
```

Reply:
Done in bc869705500dbe0aa3575b3aafd58be9507220d4